pages/_app.tsx

```diff
@@ -1,16 +1,27 @@
-// pages/_app.ts
 import '../styles/globals.css';
 
+import { MDXProvider } from '@mdx-js/react';
 import type { AppProps } from 'next/app';
+import Head from 'next/head';
+
+import Layout from '@/components/layout/Layout';
+import AppContext from '@/context/AppContext';
 
 /**
  * @description The MyApp component is the root component for the application.
  */
-function MyApp({ Component, pageProps }: AppProps) {
+function MyApp({ Component, pageProps, router }: AppProps) {
   return (
-    <div>
-      <Component {...pageProps} />
-    </div>
+    <AppContext.Provider value={{ path: router.asPath }}>
+      <Head>
+        <script async defer src='https://buttons.github.io/buttons.js'></script>
```

Comment:
Why are we adding this script tag?

Reply:
Actually this was there before too, in the master branch.